src/FortniteReplayReader/Models/GameInformation.cs

```diff
@@ -103,6 +111,36 @@ internal void UpdateLlama(uint channelId, SupplyDropLlamaC supplyDropLlama)
             newLlama.Destroyed = supplyDropLlama.bDestroyed ?? newLlama.Destroyed;
             newLlama.SpawnedItems = supplyDropLlama.bHasSpawnedPickups ?? newLlama.SpawnedItems;
         }
+        
+        internal void UpdateLabradorLlama(LabradorLlamaC labradorLlama)
+        {
+            if (!labradorLlama.PawnUniqueId.HasValue)
+            {
+                return;
+            }
+
+            var pawnUniqueId = labradorLlama.PawnUniqueId.Value;
```

Comment:
Llama changed channels for some reason…. But the unique pawn id remains the same… so let’s use that